CHANGELOG.md

```diff
@@ -10,6 +10,17 @@ nav_order: 1
 <!--## [MAJOR.MINOR.PATCH] - YYYY-MM-DD -->
 
 
+## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
+
+- Fix: services always created in the default VPC of the same cloud, when `project_vpc_id` wasn't set
+- Change `aiven_account_team_project` resource and datasource field `team_type` (enum):
+  add `organization:networking:read`, `organization:networking:write`
+- Change `aiven_kafka_native_acl` resource field `resource_type` (enum)
+- Change `aiven_organization_permission` resource field `permissions.permissions` (enum):
+  add `organization:networking:read`, `organization:networking:write`
+- Change `aiven_project_user` resource and datasource field `member_type` (enum): add `organization:networking:read`, `organization:networking:write`
+- Change `aiven_service_integration_endpoint` resource and datasource field `endpoint_type` (enum): add `external_azure_blob_storage`
```

Comment:
Turns out there is a validation. So technically, even `external_azure_blob_storage` has released its user config, user can't set the `endpoint_type` to is it, because it wasn't in the OpenAPI spec (not on the choices list).